src/snowflake/cli/_plugins/nativeapp/project_model.py

```diff
@@ -139,10 +136,13 @@ def project_identifier(self) -> str:
     @property
     def package_name(self) -> str:
         if self.definition.package and self.definition.package.name:
-            name = self.definition.package.name
+            return self.definition.package.name
```

Comment:
we need to_identifier() back in that case.

Reply:
added back